lib/browser/api/menu-item.js

```diff
@@ -6,15 +6,16 @@ nextCommandId = 0
 
 // Maps role to methods of webContents
 rolesMap = {
-  undo: 'undo',
-  redo: 'redo',
-  cut: 'cut',
-  copy: 'copy',
-  paste: 'paste',
-  selectall: 'selectAll',
-  minimize: 'minimize',
-  close: 'close',
-  delete: 'delete'
+  'undo': 'undo',
+  'redo': 'redo',
+  'cut': 'cut',
+  'copy': 'copy',
+  'paste': 'paste',
+  'paste-and-match-style': 'pasteAndMatchStyle',
```

Comment:
Current style for roles is to use all lower cases, like `hideothers` and `selectall`.

We should probably change them to lowerCamelCase in future to make the names consistent with other APIs.